atc/api/info_test.go

```diff
@@ -255,6 +256,7 @@ var _ = Describe("Pipelines API", func() {
           "vault": {
             "url": "` + credServer.URL() + `",
             "path_prefix": "testpath",
+            "lookup_templates": ["/{{.Team}}/{{.Pipeline}}/{{.Secret}}", "/{{.Team}}/{{.Secret}}"],
```

Comment:
Another side note: in the future we'd probably just call this `lookup_paths` since the prototype wouldn't really know whether or not `((vars))` were used. This would be similar to how resources don't really know whether `((vars))` are used for their various parameters in `source` and `get`/`put` steps.

It's tempting to just call this `lookup_paths` from the get-go but I don't think it matters a whole lot. It might make it easier for folks to transition to the Vault prototype configuration scheme in the future, but they'd already have to swap out the `{{.Foo}}` syntax for `((.foo))` anyway. It might be better to just leave it as a different parameter and have the prototype complain loudly about an unknown `lookup_templates` field when users switch over. :slightly_smiling_face: